lips/lip-8.md

```diff
@@ -125,12 +141,65 @@ The structure in this object can be a full pre-approval or just the fields of an
 }
 ```
 
+### ScopeObject
+
+In this object the initiator VASP declares its intent for the pre-approval, this can by one of two options:
+  1. Save the consumer sub-account for future transactions (save_sub_account)- this will enable the initiator VASP (merchant) to charge the sub-account in the future but will require the owner to approve the transaction. When using this option, amount limits are not required
+  2. Save the consumer sub-account and get a consent for future payments (consent) - this enables the initiator VASP (merchant) to charge the sub-account without any interaction with the owner. 
+Also, the scope limits the FundPullPreApprovalObject to certain parameters of time and amount. this object can be changed by the initiator VASP if needed, but any change requires the target VASP to approve the scope change.
+
+
+| Field         | Type  | Required?     | Description   |
+|-------        |------ |-----------    |-------------  |
+| Type | str enum | Y | This can be either save_sub_account or consent |
```

Comment:
I'm not sure I understand why we would want "type".  save_sub_account seems to not be overly useful, but perhaps I'm missing something.  Isn't it implied that if I wanted to bill someone again, I could try their same subaddress and they would have the option to approve it?  Even if I didn't get this approval ahead of time, I could still trigger the transaction, couldn't I?

Reply:
you are right that "consent" covers the basic option of "save_sub_account". 
but we thought it would be nice to have the user approve even the storage of their details (sub-account). 
consider the scenario of users who wants to add their Libra wallet as a payment method (to make subsequent payments easier), but still want to authorize every payment. something like a push notification to their mobile app. 
the merchant VASP should store their sub-address but without a "real" consent of withdrawing funds. 
one may argue that the merchant could save these details without prior approval of the user, but I think that users will want to authorize even this. 
the wallet VASPs on the other hand will pop a notification to the user only if the merchant could present such an approval.    

does it make sense?

Reply:
Hmmm, I'm still not convinced that this is something that merchants/users will want.  I could be wrong though.  But it feels like there is no real reason for anyone to follow this for save_sub_account.  I can just put a checkbox on my page to "save" in the same way and it functionally operates exactly the same as this without needing any off-chain or user action in their wallet app.  It just feels like we are adding a feature that isn't actually adding anything practically, but does add some complexity in implementation